init.rb

```diff
@@ -1,16 +1,20 @@
 require 'redmine'
 
+Rails.configuration.to_prepare do
+  Redmine::Activity.register 'evmbaseline'
+  Redmine::Search.available_search_types << 'evmbaseline'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.